.pre-commit-config.yaml

```diff
@@ -34,7 +34,6 @@ repos:
         args: [ --fix ]
       # Run the formatter.
       - id: ruff-format
-        args: [ --fix ]
```

Comment:
This was incorrectly added in a previous change but didn't complain for some reason.